examples/solve_generic_mpcc/generic_mpcc.m

```diff
@@ -0,0 +1,30 @@
+close all;
+clear all;
+import casadi.*;
+import nosnoc.solver.mpccsol;
+
+mpccsol_opts = struct;
+opts = nosnoc.solver.Options();
+mpccsol_opts.relaxation = opts;
+mpcc = vdx.problems.Mpcc();
+
+x = SX.sym('x', 2);
+mpcc.w.x(0) = {x,0, inf};
+mpcc.f = norm(x- [-1;0])^2;
+mpcc.G.x(0) = {x(1)};
+mpcc.H.x(0) = {x(2)};
+
+mpcc_struct = struct;
+mpcc_struct.x = mpcc.w.sym;
+mpcc_struct.g = mpcc.g.sym;
+mpcc_struct.p = mpcc.p.sym;
+mpcc_struct.G = mpcc.G.sym;
+mpcc_struct.H = mpcc.H.sym;
+mpcc_struct.f = mpcc.f;
```

Comment:
as discussed in the vdx comments,  the vdx Problem could possibly contain this struct and would shorten the manual extraction of the symbolics.